package.json

```diff
@@ -127,21 +127,21 @@
             "AzureChinaCloud",
```

Comment:
please prefer to cheery pick the comit and make a PR directly to microsoft/vscode-azure-account instead of Erics branch

Reply:
Yeah no need to PR onto my branch